anti-evil-maid/README

```diff
@@ -71,11 +60,19 @@ You should download the SINIT module required for your system.
 Intel documented the required SINIT module depending on your CPU platform in:
 http://software.intel.com/en-us/articles/intel-trusted-execution-technology
 
-You can then download the module and unzip it. All the modules can be downloaded
-from:
+You can then download the module and unzip it. All the modules can be
+downloaded from:
 
 https://software.intel.com/protected-download/267276/183305
 
+Also, make sure you have the latest RACM update, if available (2nd & 3rd gen):
+https://software.intel.com/system/files/article/183305/intel-txt-sinit-acm-revocation-tools-guide-rev1-0_2.pdf
+
+It's possible to use 3rd gen SINIT/RACM on 2nd gen platforms. In fact, the
```

Comment:
No clue, to be honest. Perhaps we should mention to use 3rd gen SINIT on 2nd gen CPUs if PCR sanity check fails (adding a "troubleshooting" readme section)?